picoquic/picoquic.h

```diff
@@ -210,6 +210,9 @@ void picoquic_free(picoquic_quic_t* quic);
 /* Set cookie mode on QUIC context when under stress */
 void picoquic_set_cookie_mode(picoquic_quic_t* quic, int cookie_mode);
 
+/* Enable the client authentication in TLS (client is required to send certificates). */
+void picoquic_enable_client_authentication(picoquic_quic_t* quic);
+
```

Comment:
How does an application disable this option if it is not needed anymore? I would prefer following the model of 
~~~
 void picoquic_set_cookie_mode(picoquic_quic_t* quic, int cookie_mode);
~~~
That is, have a an authentication_mode parameter, with value 1 to turn it on, 0 to turn it off.